src/core/execute.c

```diff
@@ -740,7 +740,7 @@ static int ask_for_confirmation(const char *vc, const char *cmdline) {
         }
 
         for (;;) {
-                r = ask_char(&c, "yfs?", "Execute %s? [y/f/s/? for help] ", e);
+                r = ask_char(&c, "yfs?i", "Execute %s? [y/f/s/? for help] ", e);
```

Comment:
"i" is missing from the question string (also I think "?" should always stay last)